src/richText.js

```diff
@@ -277,11 +277,40 @@ define([
             }
         });
 
-        // Adapted from http://www.keyvan.net/2012/11/clean-up-html-on-paste-in-ckeditor/
-        var STYLE = /<style type="text\/css">.*?<\/style>/g;
+        editor._vellum_fromRichText = function (html) {
+            return fromRichText(html, form, options.isExpression);
+        };
+
         editor.on('paste', function(event) {
-            event.data.dataValue = event.data.dataValue.replace(STYLE, "");
-        }, null, null, 2 );
+            var data = event.data;
+            if (data.dataTransfer && data.dataTransfer.getData("Text")) {
+                // Get plain text instead of HTML because HTML encoded
+                // content from applications like Word or your text
+                // editor often contains unwanted styling information.
+                //
+                // Insert HTML-ified plain text rather than HTML with
+                // bubbles because we cannot reliably find hashtags in
+                // text fragments that are not valid XPath expressions.
```

Comment:
It's more complicated than that. We have to evaluate the pasted value in the context of the expression it's being pasted into, and I don't know of a way to do that while preserving the insertion point after the pasted text. It seemed like it would take a deeper dive into CKEditor paste logic to accomplish that. Also, I found it surprisingly difficult to get bubble-ified HTML to paste into the editor properly. CKEditor's content filter was stripping parts of the bubbles out, even after I updated `CKEDITOR.config.allowedContent` to allow the relevant tags and attributes.